include/ada/url_pattern.h

```diff
@@ -218,8 +218,8 @@ class url_pattern_component {
 
  private:
   std::string pattern{};
-  std::regex_constants::syntax_option_type flags{};
-  std::regex regexp{};
+  std::regex_constants::syntax_option_type flags = std::regex::ECMAScript;
+  std::regex regexp{"*"};
```

Comment:
here I saw that `wpt_urlpattern_tests.has_regexp_groups` test passes when we use `.*` for ecmascript compatibility